Demo/AppDelegate.swift

```diff
@@ -19,7 +19,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        UIFont.registerFinniversKitFonts()
```

Comment:
Can we update the readme know that we don't need this line anymore?

Reply:
Yes :)